composer.json

```diff
@@ -36,6 +37,25 @@
   "replace": {
     "eoneopay/apiformats": "*"
   },
+  "scripts":{
+    "code-sniffer": "phpcs --standard=vendor/eonx-com/standards/php-code-sniffer/EoneoPay --colors --report=full src tests -p",
+    "copy-paste-detector": "phpcpd --ansi --min-lines=5 --min-tokens=70 src tests",
+    "mess-detector": "phpmd src,tests text phpmd.xml",
+    "phpstan": "phpstan analyze src tests -c phpstan.neon --ansi --level max --memory-limit=2048M",
+    "phpunit": "phpunit --configuration phpunit.xml",
+    "coverage": [
```

Comment:
Probably good idea to have no process timeouts for the commands that have potential to run for more than 5 minutes.

https://getcomposer.org/doc/articles/scripts.md#writing-custom-commands

In this project it wont, but as a template coverage is likely the slowest command

Reply:
I want to keep them defaulted to 5 minutes until we see where there's the need for it, and update those projects individually